docs/benchmark/ftlib_benchmark.md

```diff
@@ -0,0 +1,131 @@
+## Benchmark of FTlib AllReduce
+
+We perform experiments to test the performance of FTlib AllReduce in
+ElasticDL on Minikube, the CPU cluster, and GPU cluster. We adopt
+ResNet50 and MobileNetV2 in experiments. ResNet50 is a computation-intensive
+model and MobileNetV2 is a communication-intensive model.
+
+## Minikube
```

Comment:
The listed parameters are irrelevant to minikube, why name this section header `Minikube`?

Reply:
Yes, I should list parameters of the minikube cluster.